webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/main.js

```diff
@@ -0,0 +1,29 @@
+import './initComponents.js';
+
+// get overrided components if exists
+if (extensionRegistry) {
+  const components = extensionRegistry.loadComponents('VersionHistoryDrawer');
+  if (components && components.length > 0) {
+    components.forEach(cmp => {
+      Vue.component(cmp.componentName, cmp.componentOptions);
+    });
+  }
+}
+const appId = 'VersionHistoryDrawer';
+
+Vuetify.prototype.preset = eXo.env.portal.vuetifyPreset;
+Vue.use(Vuetify);
+
+//getting language of user
+const lang = eXo && eXo.env.portal.language || 'en';
+
+const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet.Portlets-${lang}.json`;
+
+export function init() {
+  exoi18n.loadLanguageAsync(lang, url).then(i18n => {
+    Vue.createApp({
+      i18n,
+      vuetify: Vue.prototype.vuetifyOptions,
+    }, appId, 'Version History drawer');
+  });
```

Comment:
IMO, no need to initialize an app here, all you need is to define the component in `initComponents.js` and then reuse it where it's meant to be instantiated, see `activity-reactions` module [main.js](https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/activity-reactions/main.js) file by example.